src/config/mod.rs

```diff
@@ -186,18 +187,17 @@ impl Config {
             None => {
                 format!("# CHAT:[{timestamp}]\n{input}\n--------\n{output}\n--------\n\n",)
             }
+            Some(v) if v.is_temp() => {
+                format!(
+                    "# CHAT:[{timestamp}]\n{}\n{input}\n--------\n{output}\n--------\n\n",
+                    v.prompt
+                )
+            }
             Some(v) => {
-                if v.is_temp() {
-                    format!(
-                        "# CHAT:[{timestamp}]\n{}\n{input}\n--------\n{output}\n--------\n\n",
-                        v.prompt
-                    )
-                } else {
-                    format!(
-                        "# CHAT:[{timestamp}] ({})\n{input}\n--------\n{output}\n--------\n\n",
-                        v.name,
-                    )
-                }
```

Comment:
Collapsed that `if v.is_temp() ... ` expression into the outer match expression, reduced nesting